UI/UnoEthereumBlockChain/DemoApp/DemoApp/DemoApp.Shared/MainPage.xaml

```diff
@@ -17,31 +18,17 @@
     <StackPanel CornerRadius="5"
                 Padding="1"
                 BorderThickness="1"
-                Margin="0">
-        <StackPanel.BorderBrush>
-            <media:AcrylicBrush TintOpacity="0.1"
-                                                TintLuminosityOpacity="0.9"
-                                                TintColor="{ThemeResource SystemAccentColor}"
-                                                FallbackColor="{ThemeResource SystemAccentColor}"
-                                                BackgroundSource="Backdrop"
-                                                xmlns:media="using:Windows.UI.Xaml.Media" />
-        </StackPanel.BorderBrush>
-        <StackPanel.Background>
-            <media:AcrylicBrush TintOpacity="0.7"
-                                                TintLuminosityOpacity="0.9"
-                                                TintColor="{ThemeResource SystemAccentColor}"
-                                                FallbackColor="{ThemeResource SystemAccentColor}"
-                                                BackgroundSource="Backdrop"
-                             xmlns:media="using:Windows.UI.Xaml.Media" />
-        </StackPanel.Background>
+                Margin="0"
+                BorderBrush="{StaticResource Backdrop}">        
```

Comment:
the background brush has been dropped, and there is an addition of foreground=black on the button later down
any justification on why we are doing that in a pure template update?

Reply:
I did a mistake and remove the code accidentally. The code was restored and the background was bound. with the centralized brush.

Reply:
btw, there are two different brushes: 
- one with tint=0.1 for border; and 
- one tint=0.7 for background

Reply:
Thanks, Done!